JASP-Tests/R/tests/testthat/test-RobustBayesianMetaAnalysis.R

```diff
@@ -0,0 +1,1455 @@
+context("Robust Bayesian Meta-Analysis")
+
+# path to the pre-fitted RoBMA model
+fitted_path <- "RoBMA_testfit.RDS"
```

Comment:
this needs to link to the pre-fitted model.

Reply:
Is this something Joris should do?

Reply:
anyone who knows how to make it work. The analysis needs to load the .RDS object and I have no idea how are the paths in the unit tests specified.

Reply:
Cant you just do `".\RoBMA_testfit.RDS"` ? It is right next to the unittest R file right?

Reply:
that's how I set it up in the module, but I didn't know whether the location would be the same inside of jasp-desktop

Reply:
and it's loaded from inside the `jasptools::run()` command. I'm just stating that I have no idea how this should be set-up.

Reply:
Me neither, but I figured either @TimKDJ or @vandenman would have a better guess.

Reply:
Something like `file <- file.path(jasptools:::.pkgOptions$tests.dir, "RoBMA_testfit.RDS")` should work if you place the `.RDS` file in the same directory as all the unit tests.

Reply:
it did, thanks!